### DISCUSSION TITLE
How to import glimmer/tracking?

### DISCUSSION THREAD

**Discussion by Bouke** [typed-ember/ember-cli-typescript#1457] [Q&A] [answered]
### Which package(s) does this enhancement pertain to?
  - [ ] @types/ember
  - [ ] @types/ember__string
  - [ ] @types/ember__polyfills
  - [ ] @types/ember__object
  - [ ] @types/ember__utils
  - [ ] @types/ember__array
  - [ ] @types/ember__engine
  - [ ] @types/ember__debug
  - [ ] @types/ember__runloop
  - [ ] @types/ember__error
  - [ ] @types/ember__controller
  - [ ] @types/ember__component
  - [ ] @types/ember__routing
  - [ ] @types/ember__application
  - [ ] @types/ember__test
  - [ ] @types/ember__test-helpers
  - [ ] @types/ember__service
  - [ ] @types/ember-data
  - [ ] @types/rsvp
  - [ ] Other
  - [x] I don't know

### Please write a user story for this feature

As a developer I want to import `@tracked` from `@glimmer/tracking`. I don't understand how to do that.

As a workaround I've added `glimmer.d.ts`:

```
declare module "@glimmer/tracking" {
    export function tracked(target: object, propertyKey: string | symbol): void;
}
```

**Comment by dfreeman** [comment, marked as answer]
@Bouke I believe new apps/addons include a dependency on `@glimmer/tracking` out of the box (https://github.com/ember-cli/ember-new-output/blob/2ec0339c5e1bc7ac5f23efede3dd9929262a1926/package.json#L25), which should ship with type declarations.
